ena_portal_api/ena_handler.py

```diff
@@ -200,9 +200,6 @@ def post_request(self, data):
                 self.url, data=data, auth=self.auth, **get_default_connection_headers()
             )
         else:
-            logging.warning(
-                "Not authenticated, set env vars ENA_API_USER and ENA_API_PASSWORD to access private data."  # noqa: E501
-            )
             response = requests.post(
```

Comment:
Perhaps we could refactor this as such, to have less conditionals:
 
 ```
 def post_request(self, data):
     headers = get_default_connection_headers()
     auth = {'auth': self.auth} if self.auth else {}
     response = requests.post(self.url, data=data, **auth, **headers)
     
     return response
 ```

Reply:
Yes, we will incoporate this later. Thanks!